conda_build/os_utils/liefldd.py

```diff
@@ -890,8 +890,10 @@ def get_symbols(file, defined=True, undefined=True, notexported=False, arch='nat
         except:
             pass
     res = []
+    imported_function_names = set()
     if len(binary.exported_functions):
```

Comment:
NB: Due to lack of deeper knowledge of LIEF, I have no idea what the implications are in prioritizing `binary.exported_functions` over `binary.symbols` over `binary.static_symbols`. So, IDK if it makes sense to use a (maybe?) smaller set of symbols from `.exported_functions` rather than just `.symbols`.